assets/src/features/room-page/components/SidebarTab.tsx

```diff
@@ -0,0 +1,26 @@
+import clsx from "clsx";
+import React from "react";
+import Button from "../../shared/components/Button";
+
+export type Tab = "chat" | "people";
+
+type SidebarTabProps = {
+  tab: Tab;
+  onClick: (tab: Tab) => void;
+  activeTab: Tab;
+  children?: React.ReactNode;
+};
+
+const SidebarTab: React.FC<SidebarTabProps> = ({ onClick, tab, activeTab, children }) => {
+  const isActive = tab == activeTab;
+  return (
+    <Button className={clsx(normalTabClassname, isActive && activeTabClassname)} onClick={() => onClick(tab)}>
+      {children}
+    </Button>
+  );
+};
+
+export default SidebarTab;
```

Comment:
Since `SidebarTab` probably won't be use anywhere else than within the `Sidebar` component, can it be moved to the `Sidebar.tsx` file?

Reply:
IMO it's more legible to keep it in a separate file